waterdata/utils.py

```diff
@@ -53,3 +53,55 @@ def parse_rdb(rdb_iter_lines):
     for record in rdb_iter_lines:
         record_values = record.split('\t')
         yield dict(zip(headers, record_values))
+
+
+def get_disambiguated_values(location, code_lookups, country_state_county_lookups):
+    """
+    Convert values for keys that contains codes to human readable names using the lookups
+    :param dict location:
+    :param dict code_lookups:
+    :param dict country_state_county_lookups:
+    :rtype: dict
+    """
+
+    def get_state_name (country_code, state_code):
+        """
+        Return the name of the state with country_code and state_code
+        :param str country_code:
+        :param str state_code:
+        :rtype: str
```

Comment:
It looks like this is actually returning a dictionary rather than a string.

Reply:
No it returns a string, at least it should be. It returns the state name only.

Reply:
Yep. You're right. I was looking at the returned value of the parent function. To honest, while nested functions are certainly useful, I find them difficult to read.